maven_iuvs/search.py

```diff
@@ -337,6 +339,8 @@ def get_latest_files(files):
     Prefers highest version number, then production files to stage files,
     and finally highest revision number.
 
+    NOTE: This only works if multiple versions of the files are in the list.
```

Comment:
I think if multiple versions are not present, this should just return the original list, right? So it 'works', there's just nothing to do.

Reply:
Clarified what I meant 81300aef8b8b0da8e914f3aa17b9ab02d1a8b2ab here

Reply:
Makes sense, good to clarify the docstrings here.